[init : remove] Removing all files from /workspace/source ...

[clone : clone] + '[' false '=' true ]
[clone : clone] + '[' false '=' true ]
[clone : clone] + '[' false '=' true ]
[clone : clone] + CHECKOUT_DIR=/workspace/output/
[clone : clone] + '[' true '=' true ]
[clone : clone] + cleandir
[clone : clone] + '[' -d /workspace/output/ ]
[clone : clone] + rm -rf '/workspace/output//*'
[clone : clone] + rm -rf '/workspace/output//.[!.]*'
[clone : clone] + rm -rf '/workspace/output//..?*'
[clone : clone] + test -z 
[clone : clone] + test -z 
[clone : clone] + test -z 
[clone : clone] + git config --global --add safe.directory /workspace/output
[clone : clone] + /ko-app/git-init '-url=https://github.com/VirajDalave/devops-capstone-project.git' '-revision=cd-pipeline' '-refspec=' '-path=/workspace/output/' '-sslVerify=true' '-submodules=true' '-depth=1' '-sparseCheckoutDirectories='
[clone : clone] {"level":"info","ts":1727726364.819382,"caller":"git/git.go:176","msg":"Successfully cloned https://github.com/VirajDalave/devops-capstone-project.git @ 8d5aca7c01693ce0d01716a3cf6a17b0f403b034 (grafted, HEAD, origin/cd-pipeline) in path /workspace/output/"}
[clone : clone] {"level":"info","ts":1727726364.8847222,"caller":"git/git.go:215","msg":"Successfully initialized and updated submodules in path /workspace/output/"}
[clone : clone] + cd /workspace/output/
[clone : clone] + git rev-parse HEAD
[clone : clone] + RESULT_SHA=8d5aca7c01693ce0d01716a3cf6a17b0f403b034
[clone : clone] + EXIT_CODE=0
[clone : clone] + '[' 0 '!=' 0 ]
[clone : clone] + git log -1 '--pretty=%ct'
[clone : clone] + RESULT_COMMITTER_DATE=1727726261
[clone : clone] + printf '%s' 1727726261
[clone : clone] + printf '%s' 8d5aca7c01693ce0d01716a3cf6a17b0f403b034
[clone : clone] + printf '%s' https://github.com/VirajDalave/devops-capstone-project.git

[lint : flake8] Collecting Werkzeug==2.1.2
[lint : flake8]   Downloading Werkzeug-2.1.2-py3-none-any.whl (224 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 224.9/224.9 kB 11.1 MB/s eta 0:00:00
[lint : flake8] Collecting Flask==2.1.2
[lint : flake8]   Downloading Flask-2.1.2-py3-none-any.whl (95 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 95.2/95.2 kB 45.4 MB/s eta 0:00:00
[lint : flake8] Collecting Flask-SQLAlchemy==2.5.1
[lint : flake8]   Downloading Flask_SQLAlchemy-2.5.1-py2.py3-none-any.whl (17 kB)
[lint : flake8] Collecting psycopg2-binary==2.9.3
[lint : flake8]   Downloading psycopg2_binary-2.9.3-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.0 MB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.0/3.0 MB 101.5 MB/s eta 0:00:00
[lint : flake8] Collecting python-dotenv==0.20.0
[lint : flake8]   Downloading python_dotenv-0.20.0-py3-none-any.whl (17 kB)
[lint : flake8] Collecting gunicorn==20.1.0
[lint : flake8]   Downloading gunicorn-20.1.0-py3-none-any.whl (79 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 79.5/79.5 kB 39.3 MB/s eta 0:00:00
[lint : flake8] Collecting honcho==1.1.0
[lint : flake8]   Downloading honcho-1.1.0-py2.py3-none-any.whl (21 kB)
[lint : flake8] Collecting pylint==2.14.0
[lint : flake8]   Downloading pylint-2.14.0-py3-none-any.whl (485 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 485.0/485.0 kB 9.0 MB/s eta 0:00:00
[lint : flake8] Collecting flake8==4.0.1
[lint : flake8]   Downloading flake8-4.0.1-py2.py3-none-any.whl (64 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.1/64.1 kB 26.1 MB/s eta 0:00:00
[lint : flake8] Collecting black==22.3.0
[lint : flake8]   Downloading black-22.3.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.5 MB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.5/1.5 MB 98.8 MB/s eta 0:00:00
[lint : flake8] Collecting nose==1.3.7
[lint : flake8]   Downloading nose-1.3.7-py3-none-any.whl (154 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 154.7/154.7 kB 58.0 MB/s eta 0:00:00
[lint : flake8] Collecting pinocchio==0.4.3
[lint : flake8]   Downloading pinocchio-0.4.3-py3-none-any.whl (12 kB)
[lint : flake8] Collecting factory-boy==2.12.0
[lint : flake8]   Downloading factory_boy-2.12.0-py2.py3-none-any.whl (36 kB)
[lint : flake8] Collecting coverage==6.3.2
[lint : flake8]   Downloading coverage-6.3.2-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (210 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 210.7/210.7 kB 71.9 MB/s eta 0:00:00
[lint : flake8] Collecting codecov==2.1.13
[lint : flake8]   Downloading codecov-2.1.13-py2.py3-none-any.whl (16 kB)
[lint : flake8] Collecting httpie==3.2.1
[lint : flake8]   Downloading httpie-3.2.1-py3-none-any.whl (124 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 125.0/125.0 kB 59.3 MB/s eta 0:00:00
[lint : flake8] Collecting Flask-Talisman
[lint : flake8]   Downloading flask_talisman-1.1.0-py2.py3-none-any.whl (18 kB)
[lint : flake8] Collecting Flask-Cors
[lint : flake8]   Downloading Flask_Cors-5.0.0-py2.py3-none-any.whl (14 kB)
[lint : flake8] Collecting Jinja2>=3.0
[lint : flake8]   Downloading jinja2-3.1.4-py3-none-any.whl (133 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.3/133.3 kB 48.2 MB/s eta 0:00:00
[lint : flake8] Collecting itsdangerous>=2.0
[lint : flake8]   Downloading itsdangerous-2.2.0-py3-none-any.whl (16 kB)
[lint : flake8] Collecting click>=8.0
[lint : flake8]   Downloading click-8.1.7-py3-none-any.whl (97 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 97.9/97.9 kB 41.6 MB/s eta 0:00:00
[lint : flake8] Collecting importlib-metadata>=3.6.0
[lint : flake8]   Downloading importlib_metadata-8.5.0-py3-none-any.whl (26 kB)
[lint : flake8] Collecting SQLAlchemy>=0.8.0
[lint : flake8]   Downloading SQLAlchemy-2.0.35-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 111.9 MB/s eta 0:00:00
[lint : flake8] Requirement already satisfied: setuptools>=3.0 in /usr/local/lib/python3.9/site-packages (from gunicorn==20.1.0->-r requirements.txt (line 11)) (58.1.0)
[lint : flake8] Collecting platformdirs>=2.2.0
[lint : flake8]   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
[lint : flake8] Collecting mccabe<0.8,>=0.6
[lint : flake8]   Downloading mccabe-0.7.0-py2.py3-none-any.whl (7.3 kB)
[lint : flake8] Collecting tomli>=1.1.0
[lint : flake8]   Downloading tomli-2.0.1-py3-none-any.whl (12 kB)
[lint : flake8] Collecting dill>=0.2
[lint : flake8]   Downloading dill-0.3.9-py3-none-any.whl (119 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 119.4/119.4 kB 53.9 MB/s eta 0:00:00
[lint : flake8] Collecting astroid<=2.12.0-dev0,>=2.11.5
[lint : flake8]   Downloading astroid-2.11.7-py3-none-any.whl (251 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 251.2/251.2 kB 54.3 MB/s eta 0:00:00
[lint : flake8] Collecting typing-extensions>=3.10.0
[lint : flake8]   Downloading typing_extensions-4.12.2-py3-none-any.whl (37 kB)
[lint : flake8] Collecting isort<6,>=4.2.5
[lint : flake8]   Downloading isort-5.13.2-py3-none-any.whl (92 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 92.3/92.3 kB 42.5 MB/s eta 0:00:00
[lint : flake8] Collecting tomlkit>=0.10.1
[lint : flake8]   Downloading tomlkit-0.13.2-py3-none-any.whl (37 kB)
[lint : flake8] Collecting pyflakes<2.5.0,>=2.4.0
[lint : flake8]   Downloading pyflakes-2.4.0-py2.py3-none-any.whl (69 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 69.7/69.7 kB 31.5 MB/s eta 0:00:00
[lint : flake8] Collecting mccabe<0.8,>=0.6
[lint : flake8]   Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
[lint : flake8] Collecting pycodestyle<2.9.0,>=2.8.0
[lint : flake8]   Downloading pycodestyle-2.8.0-py2.py3-none-any.whl (42 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.1/42.1 kB 16.7 MB/s eta 0:00:00
[lint : flake8] Collecting mypy-extensions>=0.4.3
[lint : flake8]   Downloading mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)
[lint : flake8] Collecting pathspec>=0.9.0
[lint : flake8]   Downloading pathspec-0.12.1-py3-none-any.whl (31 kB)
[lint : flake8] Collecting colorama
[lint : flake8]   Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
[lint : flake8] Collecting Faker>=0.7.0
[lint : flake8]   Downloading Faker-30.1.0-py3-none-any.whl (1.8 MB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 120.5 MB/s eta 0:00:00
[lint : flake8] Collecting requests>=2.7.9
[lint : flake8]   Downloading requests-2.32.3-py3-none-any.whl (64 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.9/64.9 kB 26.4 MB/s eta 0:00:00
[lint : flake8] Collecting rich>=9.10.0
[lint : flake8]   Downloading rich-13.8.1-py3-none-any.whl (241 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 241.6/241.6 kB 76.4 MB/s eta 0:00:00
[lint : flake8] Collecting defusedxml>=0.6.0
[lint : flake8]   Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
[lint : flake8] Collecting requests-toolbelt>=0.9.1
[lint : flake8]   Downloading requests_toolbelt-1.0.0-py2.py3-none-any.whl (54 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 54.5/54.5 kB 24.5 MB/s eta 0:00:00
[lint : flake8] Collecting Pygments>=2.5.2
[lint : flake8]   Downloading pygments-2.18.0-py3-none-any.whl (1.2 MB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 20.5 MB/s eta 0:00:00
[lint : flake8] Collecting multidict>=4.7.0
[lint : flake8]   Downloading multidict-6.1.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (124 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 124.1/124.1 kB 48.4 MB/s eta 0:00:00
[lint : flake8] Requirement already satisfied: pip in /usr/local/lib/python3.9/site-packages (from httpie==3.2.1->-r requirements.txt (line 29)) (23.0.1)
[lint : flake8] Collecting charset-normalizer>=2.0.0
[lint : flake8]   Downloading charset_normalizer-3.3.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (142 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 142.3/142.3 kB 63.1 MB/s eta 0:00:00
[lint : flake8] Collecting wrapt<2,>=1.11
[lint : flake8]   Downloading wrapt-1.16.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (80 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 80.1/80.1 kB 42.5 MB/s eta 0:00:00
[lint : flake8] Collecting lazy-object-proxy>=1.4.0
[lint : flake8]   Downloading lazy_object_proxy-1.10.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (67 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 67.1/67.1 kB 31.2 MB/s eta 0:00:00
[lint : flake8] Collecting python-dateutil>=2.4
[lint : flake8]   Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 229.9/229.9 kB 73.0 MB/s eta 0:00:00
[lint : flake8] Collecting zipp>=3.20
[lint : flake8]   Downloading zipp-3.20.2-py3-none-any.whl (9.2 kB)
[lint : flake8] Collecting MarkupSafe>=2.0
[lint : flake8]   Downloading MarkupSafe-2.1.5-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
[lint : flake8] Collecting idna<4,>=2.5
[lint : flake8]   Downloading idna-3.10-py3-none-any.whl (70 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 70.4/70.4 kB 38.6 MB/s eta 0:00:00
[lint : flake8] Collecting certifi>=2017.4.17
[lint : flake8]   Downloading certifi-2024.8.30-py3-none-any.whl (167 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 167.3/167.3 kB 66.1 MB/s eta 0:00:00
[lint : flake8] Collecting urllib3<3,>=1.21.1
[lint : flake8]   Downloading urllib3-2.2.3-py3-none-any.whl (126 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 126.3/126.3 kB 57.2 MB/s eta 0:00:00
[lint : flake8] Collecting PySocks!=1.5.7,>=1.5.6
[lint : flake8]   Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
[lint : flake8] Collecting markdown-it-py>=2.2.0
[lint : flake8]   Downloading markdown_it_py-3.0.0-py3-none-any.whl (87 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 87.5/87.5 kB 42.4 MB/s eta 0:00:00
[lint : flake8] Collecting greenlet!=0.4.17
[lint : flake8]   Downloading greenlet-3.1.1-cp39-cp39-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl (597 kB)
[lint : flake8]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 597.4/597.4 kB 103.2 MB/s eta 0:00:00
[lint : flake8] Collecting mdurl~=0.1
[lint : flake8]   Downloading mdurl-0.1.2-py3-none-any.whl (10.0 kB)
[lint : flake8] Collecting six>=1.5
[lint : flake8]   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[lint : flake8] Installing collected packages: nose, mccabe, honcho, Flask-Talisman, zipp, wrapt, Werkzeug, urllib3, typing-extensions, tomlkit, tomli, six, python-dotenv, PySocks, Pygments, pyflakes, pycodestyle, psycopg2-binary, platformdirs, pathspec, mypy-extensions, mdurl, MarkupSafe, lazy-object-proxy, itsdangerous, isort, idna, gunicorn, greenlet, dill, defusedxml, coverage, colorama, click, charset-normalizer, certifi, SQLAlchemy, requests, python-dateutil, pinocchio, multidict, markdown-it-py, Jinja2, importlib-metadata, flake8, black, astroid, rich, requests-toolbelt, pylint, Flask, Faker, codecov, httpie, Flask-SQLAlchemy, Flask-Cors, factory-boy
[lint : flake8] Successfully installed Faker-30.1.0 Flask-2.1.2 Flask-Cors-5.0.0 Flask-SQLAlchemy-2.5.1 Flask-Talisman-1.1.0 Jinja2-3.1.4 MarkupSafe-2.1.5 PySocks-1.7.1 Pygments-2.18.0 SQLAlchemy-2.0.35 Werkzeug-2.1.2 astroid-2.11.7 black-22.3.0 certifi-2024.8.30 charset-normalizer-3.3.2 click-8.1.7 codecov-2.1.13 colorama-0.4.6 coverage-6.3.2 defusedxml-0.7.1 dill-0.3.9 factory-boy-2.12.0 flake8-4.0.1 greenlet-3.1.1 gunicorn-20.1.0 honcho-1.1.0 httpie-3.2.1 idna-3.10 importlib-metadata-8.5.0 isort-5.13.2 itsdangerous-2.2.0 lazy-object-proxy-1.10.0 markdown-it-py-3.0.0 mccabe-0.6.1 mdurl-0.1.2 multidict-6.1.0 mypy-extensions-1.0.0 nose-1.3.7 pathspec-0.12.1 pinocchio-0.4.3 platformdirs-4.3.6 psycopg2-binary-2.9.3 pycodestyle-2.8.0 pyflakes-2.4.0 pylint-2.14.0 python-dateutil-2.9.0.post0 python-dotenv-0.20.0 requests-2.32.3 requests-toolbelt-1.0.0 rich-13.8.1 six-1.16.0 tomli-2.0.1 tomlkit-0.13.2 typing-extensions-4.12.2 urllib3-2.2.3 wrapt-1.16.0 zipp-3.20.2
[lint : flake8] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[lint : flake8] 
[lint : flake8] [notice] A new release of pip is available: 23.0.1 -> 24.2
[lint : flake8] [notice] To update, run: pip install --upgrade pip
[lint : flake8] 0

[tests : nosetests] ***** Installing dependencies *****
[tests : nosetests] Requirement already satisfied: pip in /usr/local/lib/python3.9/site-packages (23.0.1)
[tests : nosetests] Collecting pip
[tests : nosetests]   Downloading pip-24.2-py3-none-any.whl (1.8 MB)
[tests : nosetests]      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 30.2 MB/s eta 0:00:00
[tests : nosetests] Requirement already satisfied: wheel in /usr/local/lib/python3.9/site-packages (0.44.0)
[tests : nosetests] Installing collected packages: pip
[tests : nosetests]   Attempting uninstall: pip
[tests : nosetests]     Found existing installation: pip 23.0.1
[tests : nosetests]     Uninstalling pip-23.0.1:
[tests : nosetests]       Successfully uninstalled pip-23.0.1
[tests : nosetests] Successfully installed pip-24.2
[tests : nosetests] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[tests : nosetests] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
[tests : nosetests] ***** Running nosetests with: -v --with-spec --spec-color
[tests : nosetests] nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
[tests : nosetests] nose.plugins.cover: INFO: Coverage report will include only packages: ['service']
[tests : nosetests] 
[tests : nosetests] Unloadable or unexecutable test.
[tests : nosetests] 
[tests : nosetests]     A Failure case is placed in a test suite to indicate the presence of a
[tests : nosetests]     test that could not be loaded or executed. A common example is a test
[tests : nosetests]     module that fails to import.
[tests : nosetests]     
[tests : nosetests]     
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 244, in load_module
[tests : nosetests]     return load_package(name, filename)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 216, in load_package
[tests : nosetests]     return _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/sqlalchemy/__init__.py", line 294, in __getattr__
[tests : nosetests]     raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_cli_commands.py", line 8, in <module>
[tests : nosetests]     from service.common.cli_commands import db_create
[tests : nosetests]   File "/workspace/source/service/common/cli_commands.py", line 4, in <module>
[tests : nosetests]     from service import app
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/sqlalchemy/__init__.py", line 294, in __getattr__
[tests : nosetests]     raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] -------------------- >> begin captured logging << --------------------
[tests : nosetests] faker.factory: DEBUG: Not in REPL -> leaving logger event level as is.
[tests : nosetests] --------------------- >> end captured logging << ---------------------
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_models.py", line 8, in <module>
[tests : nosetests]     from service import app
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/sqlalchemy/__init__.py", line 294, in __getattr__
[tests : nosetests]     raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_routes.py", line 12, in <module>
[tests : nosetests]     from tests.factories import AccountFactory
[tests : nosetests]   File "/workspace/source/tests/factories.py", line 7, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/sqlalchemy/__init__.py", line 294, in __getattr__
[tests : nosetests]     raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] Name                             Stmts   Miss  Cover   Missing
[tests : nosetests] --------------------------------------------------------------
[tests : nosetests] service/common/__init__.py           0      0   100%
[tests : nosetests] service/common/log_handlers.py      10      8    20%   12-22
[tests : nosetests] service/config.py                   11      5    55%   12-16
[tests : nosetests] --------------------------------------------------------------
[tests : nosetests] TOTAL                               21     13    38%
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Ran 4 tests in 1.375s
[tests : nosetests] 
[tests : nosetests] FAILED (errors=4)
[tests : nosetests] 

